src/rapid_pro_to_engagement_db/configuration.py

```diff
@@ -6,10 +6,3 @@ class FlowResultConfiguration:
     flow_name: str
     flow_result_field: str
     engagement_db_dataset: str
```

Comment:
we should also move this to common/config

Reply:
I'm not sure we should - this configuration is specific to rapid pro -> db sync

Reply:
yeah sure. you can go ahead and merge.